client/pers_file_xfer.cpp

```diff
@@ -446,11 +446,13 @@ int PERS_FILE_XFER::write(MIOFILE& fout) {
     if (fxp) {
         fout.printf(
             "    <file_xfer>\n"
+            "        <estimated_xfer_time_remaining>%f</estimated_xfer_time_remaining>\n"
             "        <bytes_xferred>%f</bytes_xferred>\n"
             "        <file_offset>%f</file_offset>\n"
             "        <xfer_speed>%f</xfer_speed>\n"
             "        <url>%s</url>\n"
             "    </file_xfer>\n",
+            estimated_xfer_time_remaining(),
```

Comment:
That makes sense, but I was following the same way the estimated time remaining is coded in `results.cpp`.
https://github.com/BOINC/boinc/blob/e4fff6b935891d3b6a646344a00cd38c15f3c630/client/result.cpp#L383
Similar function, and it doesn't have its own variable either.

I need to walk the code more to verify what I was thinking, but iirc I think the purpose of this is to update the estimated time in the Manager once a second.  If this is updated on every curl event, wouldn't this value be updated more frequently then it would be displayed?  If so, isn't that wasting processing time (even if it is a small fraction of it)?

Reply:
ok, make sense. let keep it as-is now. No need to fix this part